pynxtools_xps/scienta/scienta_txt.py

```diff
@@ -66,21 +100,25 @@ def construct_data(self):
         key_map = {
```

Comment:
Also, should this not be the part of the config file?

Reply:
This is the same reasoning as in https://github.com/FAIRmat-NFDI/pynxtools-xps/pull/5#discussion_r1540956668